tests/templates/kuttl/oidc/login.py

```diff
@@ -0,0 +1,106 @@
+# $NAMESPACE will be replaced with the namespace of the test case.
+
+import logging
+import requests
+import sys
+from bs4 import BeautifulSoup
+
+logging.basicConfig(
+    level="DEBUG", format="%(asctime)s %(levelname)s: %(message)s", stream=sys.stdout
+)
+
+session = requests.Session()
+
+# Click on "Sign In with keycloak" in Airflow
```

Comment:
Can we use different text to distinguish the two instances?

Reply:
As discussed - This is a flask button, and is not necessary to override it if we write our own security manager.